rhods-operator/README.md

```diff
@@ -10,9 +10,12 @@ The current *overlays* available are for the following channels:
 * [beta](operator/overlays/beta)
 * [embedded](operator/overlays/embedded)
 * [eus-2.8](operator/overlays/eus-2.8)
+* [eus-2.16](operator/overlays/eus-2.16)
```

Comment:
EUS stable doesn't exist yet, so I would suggest we hold off on merge this portion until it is officially released.

Reply:
agreed, we should wait any changes related to 2.16 till mid-Dec.

Reply:
i think this should be fine now, by  6th Dec, self-managed users should all received  update for 2.16.0